Library/Homebrew/formula_installer.rb

```diff
@@ -625,9 +661,10 @@ def pour
       downloader.stage
     end
 
-    keg = Keg.new(formula.prefix)
-    keg.relocate_install_names Keg::PREFIX_PLACEHOLDER, HOMEBREW_PREFIX.to_s,
-      Keg::CELLAR_PLACEHOLDER, HOMEBREW_CELLAR.to_s, :keg_only => formula.keg_only?
+    if !pour_bottle? || formula.bottle.needs_relocation?
```

Comment:
Hmm, I see. Any objections to splitting that part of `relocate_install_names` into a separate method, like `relocate_text` or something similar?
Alternatively, I could bring the condition into `relocate_install_names` somehow and only block off that part, but that might be less clear.